department-of-veterans-affairs/va.gov-cms
# VACMS-6828 convert vamc service desc to rich text limited [5/3]

## Description
Slated for release mid march.
Closes #6828 . 
Blocked by #7658 
Blocked by #7662

## Testing done


## Screenshots
![image](https://user-images.githubusercontent.com/8375335/150155510-bfcf72d0-51ee-442a-b978-a09d410a3558.png)

![image](https://user-images.githubusercontent.com/8375335/150155750-76cd1db0-db8f-481b-abfb-519322d0cd79.png)

## QA steps

As cms editor i can
1. edit a VAMC system service (`node/10955/edit`) and
2. verify that the format is rich text limited 
3. Then validate Acceptance Criteria from issue
- [ ] Convert the System service description field to "Rich text limited" only 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
